syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -247,7 +248,7 @@ def reconstruct(self):
         """
         workers = self.locations
 
-        ptr_to_sh = self.wrap().send(workers[0], **no_wrap)
+        ptr_to_sh = self.copy().wrap().send(workers[0], **no_wrap)
```

Comment:
why do we need copy here?

Reply:
Destroys the original share in the due to ```.remote_get()``` (next line)